### PR TITLE
Feature/lear 112

### PR DESCRIPTION
**Pull request by rromanitc**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=777)

## Code reviewers

- [ ] @Sokolovskyi1995
- [ ] @BDubas
- [ ] @anastasiyaSht
- [ ] @rromanitc
- [ ] @martasakal
- [ ] @Zaplatynskyi
- [ ] @aberezh

### Second Level Review

- [ ] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/curriculum/models.py [hunk 1]
```diff
@@ -80,8 +80,6 @@ def create(name, owner, description='', goals=()):
         :type owner: CustomUser model
 
         :return: Curriculum object or nonenew_c = Curriculum.objects.creat(name='curr',
-         description='dededede', goals='sadsadsa', owner=11)
```
Comment (ssOleg) [inline eventually/curriculum/models.py]: hmm, what should be at the end of this line -> "Curriculum.objects.creat(name='curr', " ?

File: eventually/tests/unittests/curriculum/testmodel.py [hunk 2]
```diff
@@ -22,11 +22,11 @@ def setUp(self):
         with mock.patch('django.utils.timezone.now') as mock_time:
             mock_time.return_value = TEST_DATE
             custom_user = CustomUser.objects.create(id=1,
-                                                         email='email1@mail.com',
-                                                         password='1234',
-                                                         first_name='1fname',
-                                                         middle_name='1mname',
-                                                         last_name='1lname')
+                                                    email='email1@mail.com',
```
Comment (ssOleg) [inline eventually/tests/unittests/curriculum/testmodel.py:24]: Could you change code formatting to:
custom_user=CustomUser.objects.create(
    id=1,
    email='email1@mail.com',
    password='1234',

File: eventually/tests/unittests/curriculum/testview.py [hunk 3]
```diff
@@ -31,6 +31,15 @@ def setUp(self):
             custom_user.set_password('1111')
             custom_user.save()
 
+            custom_user_second = CustomUser.objects.create(id=2,
+                                                    email='easdmail1@mail.com',
```
Comment (ssOleg) [inline eventually/tests/unittests/curriculum/testview.py]: the same thing as above "code formatting"

File: eventually/tests/unittests/curriculum/testmodel.py (hunk 2 above)
Comment (rromanitc) [reply eventually/tests/unittests/curriculum/testmodel.py:24]: Done. Now is good?

**Review by rromanitc — COMMENTED** [review]
Done

**Review by lhalam — APPROVED** [review]

**Review by aberezh — APPROVED** [review]

**Review by martasakal — APPROVED** [review]

**Pull request merged**